code/__defines/_planes+layers.dm

```diff
@@ -238,3 +256,29 @@ GLOBAL_LIST_INIT(ghost_master, list(
 	new /obj/screen/plane_master/ghost_master(),
 	new /obj/screen/plane_master/ghost_dummy()
 ))
+
+/obj/screen/plane_master/effects_planemaster
+	appearance_flags = PLANE_MASTER | KEEP_TOGETHER
+	blend_mode = BLEND_OVERLAY
+
+/obj/screen/plane_master/effects_planemaster/openspace
+	appearance_flags = PLANE_MASTER | KEEP_TOGETHER
+	blend_mode = BLEND_OVERLAY
+
+/obj/screen/plane_master/effects_planemaster/openspace/New(newloc)
+	add_filter("openspace_blur", 0, list(type = "blur", size = 0.6))
+
+/obj/screen/plane_master/vision_cone_target
+	name = "vision cone master"
+	plane = HIDDEN_PLANE
+	render_target = "vision_cone_target"
+
+/obj/screen/plane_master/vision_cone/
```

Comment:
Having it just defined with nothing like this isn't absolutely necesarry, but if you gotta 

```suggestion
/obj/screen/plane_master/vision_cone
```